core/src/main/scala/com/rallyhealth/vapors/factfilter/data/FactOrFactSet.scala

```diff
@@ -0,0 +1,20 @@
+package com.rallyhealth.vapors.factfilter.data
+
+import cats.Order
+import cats.instances.order._
+
+import scala.collection.immutable.SortedSet
+
+final class FactOrFactSet private[FactOrFactSet] (val toSortedSet: SortedSet[Fact]) extends AnyVal
+
+object FactOrFactSet {
+
+  implicit def setOfOneFact(fact: Fact)(implicit order: Order[Fact]): FactOrFactSet = new FactOrFactSet(SortedSet(fact))
```

Comment:
These implicits seem a bit aggressive.

Reply:
This is the main purpose for this `FactOrFactSet` type. It is the magnet pattern to avoid overloading the factory methods for `FactTable` and `Evidence`. It's syntactic sugar for consistent wrapping facts (or sets of facts) as FactTables or Evidence.